content/03.installation.md

```diff
@@ -1,6 +1,61 @@
+
+
+### CNA legacy bridge requirements (optional)
+
+In order to use "old" function from the original Matlab/Octave *CellNetAnalyzer*, you need one of the following programs installed at your systems (follow the links to the programs for download and installation instructions)...
+1. [MATLAB](https://www.mathworks.de/products/matlab/index.html), **in a version from 2019b up to 2021b (older and newer versions are not supported!)**.
+2. [GNU Octave](https://www.gnu.org/software/octave/index) installed at your system, **in a version >6.4.0 (older versions are not supported!)**.
+
+...as well as the original [*CellNetAnalyzer*](http://www2.mpi-magdeburg.mpg.de/projects/cna/cna.html/).
+
+
+### Additional solvers (optional)
+
+*Note: Additional solvers are only possible if you've installed CNApy according to the first main method, i.e. by using conda*
+
+CNApy itself is already packaged and installed with the open source linear programming solver [GLPK](https://www.gnu.org/software/glpk/) which is fast enough for simple calculations.
+However, for large models and more complications calculations (such as Minimal Cut Set analyses), it is recommended to install a more sophisticated and faster solver such as [CPLEX](https://www.ibm.com/products/ilog-cplex-optimization-studio) or [Gurobi](https://www.gurobi.com/). Both of these solvers have free academic license programs.
+
+#### Connect CPLEX to CNApy
+
+*Note: This connection is only possible if you've installed CNApy according to the first main method, i.e. by using conda*
+
+Once installed CPLEX on your system (as described on [CPLEX's website](https://www.ibm.com/products/ilog-cplex-optimization-studio)), use conda in order to activate CNApy's conda environment:
+
+```bash
+conda activate cnapy-dev
+```
+
+Once you're inside the cnapy-dev environment, install the CPLEX package by using pip as follows:
+
+```bash
+(cnapy-dev) pip install cplex
+```
+
+Now, go to the folder where you installed CPLEX:
+
+```bash
+(cnapy-dev) cd your_path_to_cplex
+```
+
+Then, install CPLEX's Python API by using the following command (for further details, see [this](https://www.ibm.com/docs/en/icos/20.1.0?topic=cplex-setting-up-python-api)):
+
+```bash
+(cnapy-dev) python setup.py install
+```
+
+Now add an environmental variable called PYTHONPATH which leads the cplex Python package to CPLEX. Under Linux, you can do this as follows:
+
+```bash
+(cnapy-dev) export PYTHONPATH=your_path_to_cplex/python/your_cplex_version/your_system
+```
+
+Under Windows, you can do this by searching for your system's "environmental variables" and adding the variable PYTHONNOUSERSITE with the value "your_path_to_cplex/python/your_cplex_version/your_system" using Windows's environmental variables setting window.
```

Comment:
```suggestion
```
Also not sure if this is required...

Reply:
Yes, but only because you installed the community edition via pip before. If you skip this step and only install the API from the full CPLEX installation you should be fine